src/api/config/routes.rb

```diff
@@ -669,6 +667,7 @@ def self.public_or_about_path?(request)
 
     post 'request' => 'request#global_command'
     post 'request/:id' => 'request#request_command', constraints: cons
+    get 'request/:id/status' => 'request#status', constraints: cons
```

Comment:
This route isn't used since https://github.com/openSUSE/openSUSE-release-tools/commit/25043f9614a29244b6b7824224d0cc514883fac8 and test testcoverage for it is just fake.

IMO: R.I.P.

Reply:
@coolo why didn't you remove the route then? :angry:

Reply:
I take that as a rhetorical question